Pop-Grupa-A-Backend/app/main/controller/task_controller.py

```diff
@@ -0,0 +1,45 @@
+from flask import request
+from flask_restplus import Resource
+from ..util.DTO.ComputationTask.TaskDTO import ComputationTaskDto
+from ..services.task_service import *
+from flask_restplus import cors
+
+api = ComputationTaskDto.api
+_createModel = ComputationTaskDto.createModel
+_taskDTO = ComputationTaskDto.task
+
+
+@api.route('/')
+class TaskCreate(Resource):
+    @api.response(201, 'Task successfully created.')
+    @api.doc('create a new task')
+    @api.expect(_createModel, validate=True)
+    def post(self):
+        """Creates a new Task"""
+        data = request.json
+
+        response_message = add_task(task=data)
+        if response_message == 0:
```

Comment:
jak już zwracamy kody jako inty do controllera, to może po prostu kod statusu zwracajcie?
Jak git to return 201,, jak Bad Request to return 400. Nie trzeba będzie na poziomie serwisu skakać do controllera i patrzeć co się dzieje.
Innym sposobem jest zrobienie enuma z wartościami "BAD REQUEST, OK" itp i je zwracać.

Reply:
znalazłem w tym poradniku co bazowalem, że oni na poziomie serwisu zwracaja response na poziomie serwisu -> [user_service.py](https://github.com/cosmic-byte/flask-restplus-boilerplate/blob/master/app/main/service/user_service.py)
Spytam się jeszcze Chabra jaka według niego jest dobra praktyka.

Reply:
A możemy narazie to tak zostawić i jak sie już zmerguje wszystko to ujednolicimy to?

Reply:
jestem pewny ,że jak zostawimy to nikt już tego nie poprawi i reszta będzie też tak robić.
Myślę,że serio lepiej jak zrobicie albo 1 rozwiązanie albo już te response obiekty w serwisie też zwracać. Robota na 20-30 min.